src/elements/frame/integFluidForce.m

```diff
@@ -111,10 +111,12 @@
   % ------------ Compute drag, lift and pitch moment forces  ------------
   % The cross section fluid forces in deformed coordinates is:
   % drag cross section force vector in deformed coordinates
-  if ~isempty( constantLiftDir ) && constantLiftDir 
-      fdl =  1/2 * densityFluid * c_d * dimCharacteristic * norm( VpiRelG ) ^2 * td;
+  if ~isempty(  ILVIVBool ) && ILVIVBool
+      fdl = 1/2 * densityFluid * c_d * dimCharacteristic * norm( VpiRelGflow ) * VpiRelGflow     ;
+      fdl_il =  1/2 * densityFluid * c_d_il * p/2 * dimCharacteristic * norm( VpiRelGflow )^2 * td   ;  % U^2 along VpiRelG
```

Comment:
I suggest considering have a unique expression for the drag, including also de inline term. also. This would avoid repeating code and also avoiding have a td expression in line 116 and also a vpirel expression in 115 which (i think) are the same direction...

Reply:
we can talk about it later but I wanted to avoid vpirel because I don't want the frame velocity, just the relative direction.

Reply:
we discussed this.